src/main/java/frc/robot/subsystems/Limelight.java

```diff
@@ -0,0 +1,216 @@
+package frc.robot.subsystems;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.networktables.NetworkTableEntry;
+import edu.wpi.first.networktables.NetworkTableInstance;
+import edu.wpi.first.wpilibj.shuffleboard.BuiltInLayouts;
+import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
+import edu.wpi.first.wpilibj.shuffleboard.ShuffleboardTab;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+// TODO: Add Limelight Heartbeat Functionality ("hb" NetworkTableEntry)
+// TODO: Add Limelight JavaDocs
+// TODO: Discuss "Unused" Warning Suppression
+// TODO: Discuss Error Signalling With Default Values
+
+public class Limelight extends SubsystemBase {
```

Comment:
Is this class even necessary now that we are running through PhotonVision, or should we keep it just in-case.

Reply:
Removed.